### ISSUE TITLE
LC0010 does not work in code-workspaces

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#225]
The LC0010 documentation states the Lintercop.json needs to be part of the 'project' root.
When opening the project roots folder, the LC0010 is calculated correctly.

However, when opening a code-workspace file (all our apps have a workspace with an alpackages shortcut, App folder and Test folder), the LC0010 does **no** longer respect the settings from the individual app / project folders.

![image](https://user-images.githubusercontent.com/25268332/195274313-15cfc98c-c24e-4ee8-b500-9a26945f178c.png)

**Comment by StefanMaron** [comment]
Just tried to reproduce, and it works for me. I can even have individual LinterCop.json files for each workspace folder

**Comment by fvet** [comment]
I did reload and I did not work.
Then I opened the project root folder instead.
Reopened the workspace again and it did work.

WIll keep an eye on it and re-open if needed / if I can better simulate an off-scenario.

Thx!

**Comment by fvet** [comment]
Hey Stefan.

Please find below a setup of our workspaces.

![image](https://user-images.githubusercontent.com/25268332/197811686-b15ac029-9ac6-40f9-844e-65eb451e7868.png)

The code-workspace contains various folders:

```
"folders": [       
        {
            "path": "../.alpackages"
        }, 
        {
            "path": "../Dynavision Core/App",
            "name": "ESCA - Core"
        },        
        {
            "path": "App",
            "name": "ESCN - Purchasing and Procurement Center"
        },
        {
            "path": "Test",
            "name": "ESCN - Purchasing and Procurement Center (Test)"
        }
    ],
```

... of which only one (the `/App` folder) contains a `LinterCop.json` file

{
    "cyclomaticComplexetyThreshold": 25,
    "maintainablityIndexThreshold": 20,
    "enableRule0011ForTableFields": false
}

Still files under the `/App` folder result in LC0010 warnings, where they should not, since the `cyclomaticComplexetyThreshold` has been increased to 25 for these files.

**Comment by Arthurvdv** [comment]
If I'm not mistaken @StefanMaron has resolved this in [v0.29.6.](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.29.6.).

@fvet and/or @ernestasjuska Could you confirm this?

**Comment by fvet** [comment]
@Arthurvdv I'm currently not using LC0010 pro-actively, however I just tested the reported scenario again in my workspace and can confirm that all works, so I'll be closing the issue ;)
